hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/StaleRecoveringContainerScrubbingService.java

```diff
@@ -84,9 +84,8 @@ static class RecoveringContainerScrubbingTask implements BackgroundTask {
 
     @Override
     public BackgroundTaskResult call() throws Exception {
-      containerSet.getContainer(containerID).delete();
-      containerSet.removeContainer(containerID);
-      LOG.info("Delete stale recovering container {}", containerID);
+      containerSet.getContainer(containerID).markContainerUnhealthy();
```

Comment:
@errose28 could you please check if you have any concerns if any container moved to unhealthy randomly. 
This container was in RECOVERING state and moved to unhealthy.